### DISCUSSION TITLE
Are more small caps planned?

### DISCUSSION THREAD

**Discussion by MattMayuga** [rsms/inter#587] [Q&A] [answered]
![image](https://github.com/rsms/inter/assets/26103475/6663821e-5cb0-4145-b3fb-9c06db897fb0)

As far as I know, Glyphs, the program used to make the font, autogenerates the feature, and the glyphs present are mapped to codepoints and/or used in compositions. In the image above, Inter Variable version 4.0 beta 9g is used. Here, not all letters are changed to small caps. Also, the small capital I shouldn't have serifs.

I would like to know if the small capitals will be completed some time after the release of version 4.0 (this is my prediction).

**Comment by TaigaYamada** [comment, marked as answer]
Small caps is requested in #32
